stable/datadog/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: datadog
-version: 1.31.2
+version: 1.31.3
```

Comment:
It looks more like a minor version bump than a patch one to me, WDYT?

Reply:
Right, I'll make `1.32.0`